pkg/liqoctl/offload/namespace.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019-2021 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package offload
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	offloadingv1alpha1 "github.com/liqotech/liqo/apis/offloading/v1alpha1"
+	"github.com/liqotech/liqo/pkg/consts"
+	"github.com/liqotech/liqo/pkg/liqoctl/common"
+	argsutils "github.com/liqotech/liqo/pkg/utils/args"
+	logsutils "github.com/liqotech/liqo/pkg/utils/logs"
+)
+
+func HandleOffloadCommand(ctx context.Context, command *cobra.Command, args []string) error {
+	if !klog.V(4).Enabled() {
+		klog.SetLogFilter(logsutils.LogFilter{})
+	}
+
+	config, err := common.GetLiqoctlRestConf()
+	if err != nil {
+		return err
```

Comment:
I would log something in case of errors. Same thing for the cases below.

Reply:
The returned error will be always logged, similarly to other logs. We should maybe add a test.